README.md

```diff
@@ -24,7 +24,7 @@ EN | [简中](README_zh-CN.md) | [繁中](README_zh-TW.md) | [FR](README_fr-FR.m
 
 ### Requirements
 
-* Java SE - 17 ([link](https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html))
+* Java SE - 17 ([link](https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html ))
```

Comment:
This is not necessary with markdown and there are many more links in this file which do not add a space at the end of the link.